interop/tapir/src/test/scala/caliban/interop/tapir/TapirAdapterSpec.scala

```diff
@@ -232,7 +232,7 @@ object TapirAdapterSpec {
               test("Returns 400 status code on validation errors") {
                 def test_(method: String) =
                   for {
-                    resp  <- runHttpRequest(method, acceptHeader, query = "{characterss { name }}")
+                    resp  <- runHttpRequest(method, acceptHeader, query = "{characters_ { name }}")
```

Comment:
I'm assuming this is just trying to validate for something other than `characters` and I find using a `_` is sufficient proof of that, while allowing a spell checker to identify other real typos...